load-data-to-bigquery/README.md

```diff
@@ -0,0 +1,30 @@
+## Steps to run the sample
+
+### Configuring Big Query
```

Comment:
Shall we rephrase these as steps?
1. Configure Big Query Data warehouse
2. Configure Config.toml
3.....

Reply:
Updated in [28a3a](https://github.com/ballerina-guides/etl-samples/pull/6/commits/28a3a442c39b15722882767eca520e6a04bb269e).